### PR TITLE
[CPS-2559] fix(type): add daemonset updateStrategy type

### PR DESCRIPTION
**Pull request by whalecold**
**What this PR does / why we need it**:

 fix(type): add daemonset updateStrategy type

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by whalecold** [comment]
@hezhizhen

**Comment by whalecold** [comment]
@zxq-bit

**Comment by whalecold** [comment]
把一个不该要的文件上传了

File: templates/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+apiVersion: extensions/v1beta1
```
Comment (zxq-bit) [inline templates/daemonset.yaml]: app？

Comment (whalecold) [reply templates/daemonset.yaml]: 这个文件不需要的 误传

**Pull request merged**